remoteappmanager/application.py

```diff
@@ -205,7 +205,13 @@ def _reverse_proxy_init(self):
 
     def _db_init(self):
         """Initializes the database connection."""
-        self.db = orm.AppAccounting(self.file_config.db_url)
+        if self.file_config.db_url.endswith('.db'):
+            self.db = orm.AppAccounting(self.file_config.db_url)
+        elif self.file_config.db_url.endswith('.csv'):
+            self.db = csv_db.CSVAccounting(self.file_config.db_url)
```

Comment:
Know that it's temporary, but I would check if it's a sqlalchemy path vs filesystem path first.